### PR TITLE
chore: updated design system dependencies

### PR DESCRIPTION
**Pull request by tft-abel**

### CODE REVIEW

File: ios/Podfile.lock
Comment (reidzeibel) [inline ios/Podfile.lock:1]: I think this change should not be here, since it shouldn't be related to the changes in `packages.json` and `yarn.lock`

**Review by reidzeibel — CHANGES_REQUESTED** [review]

File: ios/Podfile.lock
Comment (jorelosorio) [reply ios/Podfile.lock:1]: The podfile yes, seems to be wrong.

Comment (tft-abel) [reply ios/Podfile.lock:1]: I had a feeling

**Review by reidzeibel — APPROVED** [review]
🚀

**Pull request merged**